ENIGMAsystem/SHELL/Graphics_Systems/None/NONEModelStruct.h

```diff
@@ -16,29 +16,27 @@
 *** You should have received a copy of the GNU General Public License along
 *** with this code. If not, see <http://www.gnu.org/licenses/>
 **/
-#include "../General/GSd3d.h"
-#include "../General/GSmodel.h"
-#include "../General/GSprimitives.h"
+#include "Graphics_Systems/General/GSd3d.h"
```

Comment:
Not necessarily now, but in a future PR, could you add an `#ifdef INCLUDED_FROM_SHELLMAIN #error` pattern here? I feel like this file in particular needs one. Call it intuition.

Reply:
Yes, I can, and I actually wanted to too but was unsure. Now that you've suggested this, I know it's the right thing to do. I will, in a future pr, cover a lot of these headers and do just exactly that.